app/models/chargeback_rate_detail.rb

```diff
@@ -102,27 +102,22 @@ def hourly(rate, consumption)
     hourly_rate
   end
 
-  # Scale the rate in the unit difine by user to the default unit of the metric
-  # It showing the default units of the metrics:
-  # cpu_usagemhz_rate_average --> megahertz
-  # derived_memory_used --> megabytes
-  # derived_memory_available -->megabytes
-  # net_usage_rate_average --> kbps
-  # disk_usage_rate_average --> kbps
-  # derived_vm_allocated_disk_storage --> bytes
-  # derived_vm_used_disk_storage --> bytes
+  # Scale the rate in the unit defined by user -> to the default unit of the metric
+  MetricUnits = {
```

Comment:
A man can learn many languages. But my brain does not scale to remember all the naming conventions. :-(

Thanks for keeping an eye on me, when rubocop is not.